adagrams/game.py

```diff
@@ -1,11 +1,145 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    letter_pool_list = []
+    for letter in LETTER_POOL:
+        for freq in range(LETTER_POOL[letter]):
+            letter_pool_list.append(letter)
```

Comment:
One suggestion that I have here maybe turning this section into one loop! That way we can have better time complexity than what we have here with our nested loops, an implementation could look something like this:

```python
letters = []
for letter in LETTER_POOL: # this will give us the keys
   letters += letter * LETTER_POOL[letter]
```